packages/quantic/force-app/main/default/lwc/quanticRecommendationList/quanticRecommendationList.css

```
@@ -0,0 +1,12 @@
+.recommendations-list__container {
+  flex-wrap: wrap;
+}
+
+.recommendation-item__container {
+  width: var(--recommendationItemWidth, 33.33%);
```

I think this would be more precise, calc() is widely [supported](https://caniuse.com/calc)

```suggestion
  width: var(--recommendationItemWidth, calc(100% / 3));
```

---

I heard some rumours about `css calc()` being relatively slow, thus I'm a bit against using it here, especially that's it's not crucially necessary here since it's just a fallback value that will very rarely be used.

---

fair point, at the end of the day not of significant impact either way 👍